e2e/cypress/support/index.ts

```diff
@@ -14,7 +14,16 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import './commands'
+import './commands';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+before(function () {
+  if (Cypress.env('ADMIN_TOKEN') === undefined || Cypress.env('ADMIN_TOKEN') === '') {
```

Comment:
Consider simplifying to
```suggestion
  if (!Cypress.env('ADMIN_TOKEN')) {
```